src/App.js

```diff
@@ -44,67 +68,189 @@ const lightTheme = {
-            <StyledImg alt="Coin" src={props.coins.coins[0]?.image} />
-            <GlobalInfo>{(formatPercent(props.main.globalInfo.data?.market_cap_percentage.btc.toFixed(0)).toString())}{" "}<ProgressBar progress={props.main.globalInfo.data?.market_cap_percentage.btc} width={"50px"} height={"13px"} /></GlobalInfo>
-            <StyledImg alt="Coin" src={props.coins.coins[1]?.image} />
-            <GlobalInfo>{(formatPercent(props.main.globalInfo.data?.market_cap_percentage.eth.toFixed(0)).toString())}{" "}<ProgressBar progress={props.main.globalInfo.data?.market_cap_percentage.eth} width={"50px"} height={"13px"} /></GlobalInfo>
+            <GlobalInfoOne>
+              Conis {props.main.globalInfo.data?.active_cryptocurrencies}
+            </GlobalInfoOne>
+            <GlobalInfoTwo>
+              &#x2022; {props.main.symbol}
+              {formatNum(
+                props.main.globalInfo.data?.total_market_cap[
+                  `${props.main.currentCurrency}`
+                ].toString()
+              )}
+            </GlobalInfoTwo>
+            <GlobalInfoThree>
+              &#x2022; {props.main.symbol}
+              {formatNum(
+                props.main.globalInfo.data?.total_volume[
+                  `${props.main.currentCurrency}`
+                ].toString()
+              )}{" "}
+              <ProgressBar
+                progress={
+                  (props.main.globalInfo.data?.total_volume[
+                    `${props.main.currentCurrency}`
+                  ] /
+                    props.main.globalInfo.data?.total_market_cap[
+                      `${props.main.currentCurrency}`
+                    ]) *
+                  100
+                }
+                width={"50px"}
+                height={"13px"}
+              />
+            </GlobalInfoThree>
+            <StyledImgBC alt="Coin" src={props.coins.coins[0]?.image} />
+            <GlobalInfoFour>
+              {formatPercent(
+                props.main.globalInfo.data?.market_cap_percentage.btc.toFixed(0)
+              ).toString()}{" "}
+              <ProgressBar
+                progress={props.main.globalInfo.data?.market_cap_percentage.btc}
+                width={"50px"}
+                height={"13px"}
+              />
+            </GlobalInfoFour>
+            <StyledImgEth alt="Coin" src={props.coins.coins[1]?.image} />
+            <GlobalInfoFive>
+              {formatPercent(
+                props.main.globalInfo.data?.market_cap_percentage.eth.toFixed(0)
+              ).toString()}{" "}
+              <ProgressBar
+                progress={props.main.globalInfo.data?.market_cap_percentage.eth}
+                width={"50px"}
+                height={"13px"}
```

Comment:
do you use this progress bar with a different height? if no, consider removing this prop

Reply:
I switch the height in other areas but I also have a default height